test/specs/mainPage/equipment.ts

```diff
@@ -0,0 +1,86 @@
+import { expect } from "@wdio/globals";
+import MainPage from "../../pageobjects/main.page.ts";
+import UnitPage from "../../pageobjects/unit.page.ts";
+import ProductsPage from "../../pageobjects/products.page.ts";
+import * as testData from "../../data/testdata.json";
+
+describe("Rentzila", () => {
+  it('C213- Checking ""Спецтехніка"" section on the main page', async () => {
+    await MainPage.open();
+
+    await expect(MainPage.equipmentTitle).toBeExisting();
+    await expect(MainPage.equipmentTitle).toHaveText("Спецтехніка");
+
+    await verifyEquipment(
+      MainPage.popularEquipmentLabel,
+      testData.equipment.popular.names,
+      testData.equipment.popular.namesInFilters,
+      testData.equipment.popular.namesInUnitPage
+    );
+    await verifyEquipment(
+      MainPage.agriculturalEquipmentLabel,
+      testData.equipment.agricultural.names,
+      testData.equipment.agricultural.namesInFilters,
+      testData.equipment.agricultural.namesInUnitPage
+    );
+    await verifyEquipment(
+      MainPage.buildingEquipmentLabel,
+      testData.equipment.building.names,
+      testData.equipment.building.namesInFilters,
+      testData.equipment.building.namesInUnitPage
+    );
+    await verifyEquipment(
+      MainPage.otherEquipmentLabel,
+      testData.equipment.other.names,
+      testData.equipment.other.namesInFilters,
+      testData.equipment.other.namesInUnitPage
+    );
+  });
+  /**
+   * Verifies the equipment names, filter names, and categories by interacting with the UI elements.
+   *
+   * This function performs the following actions:
```

Comment:
@Anastas11aZakharova Good job! But, I think, the description above will be enough, the main things are the methods description and parameters description. But nothing to change here, everything is great!